src/tests/IHaskell/Test/Eval.hs

```diff
@@ -173,11 +157,37 @@ testEval =
 #else
       ":typ 3" `becomes` ["3 :: forall t. Num t => t"]
 #endif
+
+    it "evaluates :k directive" $ do
       ":k Maybe" `becomes` ["Maybe :: * -> *"]
+
+    it "evaluates :in directive" $ do
 #if MIN_VERSION_ghc(8,10,0)
-      ":in String" `pages` ["type String :: *\ntype String = [Char]\n  \t-- Defined in \8216GHC.Base\8217"]
+      displayDatasBecome ":in String" [
+        ManyDisplay [Display [
+                        DisplayData PlainText "type String :: *\ntype String = [Char]\n  \t-- Defined in \8216GHC.Base\8217"
+                        , DisplayData MimeHtml "<div class=\"code CodeMirror cm-s-jupyter cm-s-ipython\"><span class=\"cm-keyword\">type</span><span class=\"cm-space\"> </span><span class=\"cm-variable-2\">String</span><span class=\"cm-space\"> </span><span class=\"cm-atom\">::</span><span class=\"cm-space\"> </span><span class=\"cm-atom\">*</span><span class=\"cm-space\"><br /></span>\n<span class=\"cm-keyword\">type</span><span class=\"cm-space\"> </span><span class=\"cm-variable-2\">String</span><span class=\"cm-space\"> </span><span class=\"cm-atom\">=</span><span class=\"cm-space\"> </span><span class=\"cm-atom\">[</span><span class=\"cm-variable-2\">Char</span><span class=\"cm-atom\">]</span><span class=\"cm-space\"><br />  \t</span><span class=\"cm-comment\">-- Defined in \8216GHC.Base\8217</span><span class=\"cm-space\"><br /></span></div>"
+                        ]]
+        ]
+#elif MIN_VERSION_ghc(8,4,0)
+      displayDatasBecome ":in String" [
+        ManyDisplay [Display [
+                        DisplayData PlainText "type String = [Char] \t-- Defined in `GHC.Base'"
```

Comment:
It seems like most of the remaining test failures are because it's some gross unicode instead of the quotes around `GHC.Base`.

Reply:
Yeah, I suspect it depends on locale and that's why it passes in my local Docker container.

Reply:
I don't know what the correct solution is here. We could explicitly specify the locale as part of the workflow, but then the tests would fail locally, which isn't what we want. Maybe we could try using [`setLocaleEncoding`](https://hackage.haskell.org/package/base-4.18.0.0/docs/GHC-IO-Encoding.html#v:setLocaleEncoding)? IMHO that's pretty janky also.

Reply:
Hmm, I think doing `setLocaleEncoding` to put the locale in a known state within the tests is an okay thing to do.

It leaves the possibility that users with different locales will get different results. But if any of those results are bad somehow then another test suite could be added which tests different locales.

Reply:
I agree, I think for our purposes it will work fine.

Reply:
Unfortunately `setLocaleEncoding` doesn't seem to work (see my experiments below).

It seems the path of least resistance is just to just assume a UTF-8 locale, and expect UTF quotes in the tests. If a dev environment has a different locale for some reason, you can set `LC_ALL=C.UTF-8`.

With that change the eval tests are passing and we're almost there!

Reply:
It's green :tada: